tuning/data/setup_dataprocessor.py

```diff
@@ -396,10 +396,6 @@ def process_dataargs(
         is_padding_free=is_padding_free,
     )
 
-    dataset_kwargs = {}
-    if is_tokenized_dataset:
-        dataset_kwargs["skip_prepare_dataset"] = True
-
```

Comment:
Any reason we dont want to skip anymore?

Reply:
yes...added explanation here https://github.com/foundation-model-stack/fms-hf-tuning/pull/468#issue-2850878477
We need to remove this check to call `prepare_dataset` and and enable packing for the pretokenized dataset.

Reply:
Just checked, looks like it checks for tokenized with trl and skips prepare.

Reply:
This line is what we need - https://github.com/huggingface/trl/blob/a34987956cd5bf08ed7501da2510b9404bede695/trl/trainer/sft_trainer.py#L467